satpy/readers/fci_l2_nc.py

```diff
@@ -401,3 +401,58 @@ def _modify_area_extent(stand_area_extent):
         area_extent = tuple([ll_x, ll_y, ur_x, ur_y])
 
         return area_extent
+
+class FciL2NCAMVFileHandler(FciL2CommonFunctions, BaseFileHandler):
+    """Reader class for FCI L2 AMV products in NetCDF4 format."""
+    def __init__(self, filename, filename_info, filetype_info):
+        """Open the NetCDF file with xarray and prepare for dataset reading."""
+        super().__init__(filename, filename_info, filetype_info)
+
+        # Use xarray's default netcdf4 engine to open the file
+        self.nc = xr.open_dataset(
+            self.filename,
+            decode_cf=True,
+            mask_and_scale=True,
+            chunks={
+                "number_of_images": CHUNK_SIZE,
+                # 'number_of_height_estimates': CHUNK_SIZE,
+                "number_of_winds": CHUNK_SIZE
+            }
+        )
+
+    def _get_global_attributes(self):
+        """Create a dictionary of global attributes to be added to all datasets.
+
+        Returns:
+            dict: A dictionary of global attributes.
+                filename: name of the product file
+                spacecraft_name: name of the spacecraft
+                sensor: name of sensor
+                platform_name: name of the platform
+
+        """
+        attributes = {
+            "filename": self.filename,
+            "spacecraft_name": self.spacecraft_name,
```

Comment:
are both `spacecraft_name` and `platform_name` needed?

Reply:
well I assume once we get more than one MTG-I in orbit and operation maybe so.

Reply:
ok.